microservices/ga/datamgr/cheatsheet.md

```diff
@@ -158,7 +158,7 @@ status: {}
 $ kubectl create -f dnsutil.yaml 
 pod/dnsutils created
 
-$ kubectl get po -o wide
+$ kubectl get pod -o wide
```

Comment:
same as above, I was using short names for all the examples so I think it is consistent